src/main/java/twistedgate/immersiveposts/client/model/PostBaseLoader.java

```diff
@@ -33,15 +27,11 @@ public PostBaseModelRaw read(JsonObject jsonObject, JsonDeserializationContext d
 	}
 	
 	public static class PostBaseModelRaw implements IUnbakedGeometry<PostBaseModelRaw>{
+
 		@Override
-		public BakedModel bake(IGeometryBakingContext context, ModelBakery bakery, Function<Material, TextureAtlasSprite> spriteGetter, ModelState modelState, ItemOverrides overrides, ResourceLocation modelLocation){
+		public BakedModel bake(IGeometryBakingContext context, ModelBaker baker, Function<Material, TextureAtlasSprite> spriteGetter, ModelState modelState, ItemOverrides overrides, ResourceLocation modelLocation) {
 			return new PostBaseModel();
 		}
-		
-		@Override
-		public Collection<Material> getMaterials(IGeometryBakingContext context, Function<ResourceLocation, UnbakedModel> modelGetter, Set<Pair<String, String>> missingTextureErrors){
-			return ImmutableList.of();
-		}
```

Comment:
Wait.. was the `MC-1.19.2` branch used? Why didnt you use the `MC-1.19.2` branch? It is afterall the "latest" one and is quite different to the `MC-1.19.2` branch. Even if it is not the main branch.

Knowing this now makes me realize i don't have to keep reviewing this.

Reply:
I did use 1.19.2 as a reference. I'm sorry, didn't see the 1.19.3 branch